code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -17,16 +17,15 @@
 	var/agonyforce = 40
 	var/status = FALSE		//whether the thing is on or not
 	var/hitcost = 100
-	var/obj/item/weapon/cell/cell
-	var/obj/item/weapon/cell/starting_cell = /obj/item/weapon/cell/medium/high
+	var/obj/item/weapon/cell/cell = /obj/item/weapon/cell/medium/high
```

Comment:
uhm, nice work but not what i expected and this is out of step with the rest of the code. would you mind rolling back this commit? and I apologize for the inconvenience.

Reply:
uhh sure, what did you want out of curiosity? like
```suggestion
	var/obj/item/weapon/cell/cell = new /obj/item/weapon/cell/medium/high()
```
It works but I can't find anywhere else in the code that does something like that out of a proc so I wasn't sure if people were avoiding it for some reason